### PR TITLE
fix for naws dropdown sorting by key instead of value

### PR DESCRIPTION
**Pull request by pjaudiomv**
the last fix was silly, this one may be too. But i was sorting by key and really needed to sort by value. Because the default dropdown is the top one and that key is null. im shifting it off sorting and then unshifting it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ CHANGELIST
 - Rename "Get A NAWS Format Dump" to "Get A NAWS Format Export" in the semantic workshop
 - Fixed an issue where the admin user interface could falsely think a meeting had been edited
 
+
```
Comment (jbraswell) [inline README.md]: I think you've got an extra line here

File: main_server/local_server/server_admin/c_comdef_admin_main_console.class.php [hunk 2]
```diff
@@ -381,7 +381,9 @@ public function return_main_console_html()
                 $ret .= 'var g_naws_popup_prompt = \''.self::js_html($this->my_localized_strings['comdef_server_admin_strings']['world_format_codes_prompt']).'\';'.(defined('__DEBUG_MODE__') ? "\n" : '');
                 $ret .= "var g_naws_values = [";
                     $n_first = true;
-            ksort($this->my_localized_strings['comdef_server_admin_strings']['world_format_codes']);
```
Comment (jbraswell) [inline main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: This looks good, but it'd be helpful to put a comment in the code explaining why you're doing what you're doing

Comment (pjaudiomv) [reply main_server/local_server/server_admin/c_comdef_admin_main_console.class.php]: ok I added notes.

File: README.md [hunk 3]
```diff
@@ -62,6 +62,7 @@ CHANGELIST
 - Changed JSONP mime type to `application/javascript`
 - Rename "Get A NAWS Format Dump" to "Get A NAWS Format Export" in the semantic workshop
 - Fixed an issue where the admin user interface could falsely think a meeting had been edited
+- Fix for NAWS format drop down not sorting correctly.
```
Comment (jbraswell) [inline README.md]: “Fixed an issue where <describe the issue>”

**Pull request merged**